src/routes/Docs/components/Sidebar/Sidebar.scss

```diff
@@ -22,85 +18,91 @@ $border-color: #d9e6f1;
     background-color: $background-color;
   }
 
-  > ul {
-    padding: 30px 0;
+  .scrollbar {
     position: sticky;
     top: 60px; // The header's height
-    overflow: auto;
-    max-height: calc(100vh - 100px);
 
-    a {
-      display: flex;
-      align-items: center;
+    @include media-max('sm') {
```

Comment:
Why did you move this here? We want the entire sidebar to be hidden, not just the scrollbar